README.md

```diff
@@ -7,9 +7,34 @@ Azure ResourceManager Exporter
 
 Prometheus exporter for Azure Resources and information.
 
-Configuration
+Installation
 -------------
 
+[Helm 3](https://helm.sh/) must be installed to use the chart. Once Helm is set up properly, add the repo as follows:
+
+```shell
+$ helm repo add azure-resourcemanager-exporter https://carrefour-group.github.io/azure-resourcemanager-exporter
+$ helm repo update
+```
+
+Install Chart
+-------------
+The chart can be installed as follows:
+```shell
+$ helm install [RELEASE_NAME] azure-resourcemanager-exporter/azure-resourcemanager-exporter
+```
+
+Uninstall Chart
+-------------
+To uninstall the chart 
+
+```shell
+$ helm uninstall [RELEASE_NAME] 
+```
+
+Exporter Configuration
+---------------------
+
 Normally no configuration is needed but can be customized using environment variables.
 
 | Environment variable              | DefaultValue                | Description                                                       |
```

Comment:
Can you add a `Contributing` section with a `Release` subsection that mention what needs to be bumped in order to release correctly?
helm `appVersion`, `version` and helm values `image.tag` should be bumped